.pre-commit-config.yaml

```diff
@@ -0,0 +1,12 @@
+# Optional local setup for a git pre-commit hook to automatically format/lint.
+# This avoids sending a red PR and having to find the buildifier output on the CI results page.
+# See https://pre-commit.com for more information on installing pre-commit.
+# See https://pre-commit.com/hooks.html for more hooks.
+
+repos:
+    # Check formatting and lint for starlark code
+    - repo: https://github.com/keith/pre-commit-buildifier
```

Comment:
Cc @keith, can we move this into a bazelbuild? If this is a recommended way I believe it’s better that it’s community owned.

Reply:
yea I'm open to it (I assume you mean bazel-contrib), but haven't chatted with anyone about it